### PR TITLE
Fix partitioning proposal for BTRFS on aarch64 (bsc#974280)

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by mvidner** [comment]
Looks Good To Me

File: package/yast2-storage.changes [hunk 1]
```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
+Sun Apr 10 18:27:50 UTC 2016 - dmueller@suse.com
+
+- raise min /boot size for BTRFS/aarch64 (bsc#974280)
```
Comment (imobach) [inline package/yast2-storage.changes]: Please, add the version number.

Comment (mvidner) [inline package/yast2-storage.changes]: I did so, in the merge commit.

**Comment by mvidner** [comment]
Thank you, @dirkmueller !

File: src/modules/Partitions.rb [hunk 2]
```diff
@@ -225,7 +225,11 @@ def BootSizeK
         }
         Ops.set(@boot_size_k, :proposed, 150 * 1024) if EfiBoot()
 
-        if Arch.ia64
+        if Arch.aarch64
```
Comment (k0da) [inline src/modules/Partitions.rb]: POWER is 64k too

**Pull request merged**